ios/FluentUI/TwoLineTitleView/TwoLineTitleView.swift

```diff
@@ -57,11 +59,23 @@ open class TwoLineTitleView: UIView {
         case dark
     }
 
+    @objc(MSFTwoLineTitleViewAlignment)
+    public enum Alignment: Int {
+        case center
+        case leading
+    }
+
     @objc(MSFTwoLineTitleViewInteractivePart)
     public enum InteractivePart: Int {
-        case none
-        case title
-        case subtitle
+        // The @objc requirement doesn't let us use OptionSet, so we provide the bitmasks and the `contains` method ourselves
+        case none = 0
+        case title = 0b01
+        case subtitle = 0b10
+        case all = 0b11
+
+        func contains(_ other: InteractivePart) -> Bool {
```

Comment:
similar to optionset, do we need helper function of  add and remove these conditions for objc clients?

Reply:
Unfortunately this would require turning `InteractivePart` into a class that extends `NSObject`, which is a LOT more heavyweight than necessary. I want to rework `TwoLineTitleView` anyway to be simpler to use, anyway, so I don't think this is worth sweating at the moment.